script.js

```diff
@@ -0,0 +1,65 @@
+const roundingResult = 0.1 + 0.2
+console.log(roundingResult.toFixed(1));
```

Comment:
.toFixed() is going to return the result as a string, so you need to convert it to number `+roundingResult.toFixed(1)`